packages/grafana-ui/src/components/Table/types.ts

```diff
@@ -84,9 +80,31 @@ export interface Props {
   footerOptions?: TableFooterCalc;
   footerValues?: FooterItem[];
   enablePagination?: boolean;
-  cellHeight?: TableCellHeight;
+  cellHeight?: schema.TableCellHeight;
   /** @alpha */
   subData?: DataFrame[];
   /** @alpha Used by SparklineCell when provided */
   timeRange?: TimeRange;
 }
+
+export interface CustomCellRendererProps {
+  field: Field;
+  index: number;
```

Comment:
is this fieldIndex or rowIndex? I think it would be good to be clear here.

Reply:
+1

Reply:
yep it is `rowIndex` will change